capabilities/metrics-server/zarf.yaml

```diff
@@ -1,11 +1,18 @@
 kind: ZarfPackageConfig
 metadata:
-  name: example
-  description: "Example description"
+  name: uds-capability-metrics-server
+  description: "UDS Metrics Server"
+  url: https://github.com/kubernetes-sigs/metrics-server
 
 components:
-  - name: test
-    actions:
-      onCreate:
-        after:
-          - cmd: echo "What can Doug do for you?"
+  - name: metrics-server
+    required: true
+    charts:
+      - name: metrics-server
+        namespace: metrics-server
+        url: https://kubernetes-sigs.github.io/metrics-server
+        version: 3.11.0
+        valuesFiles:
+          - "values/values.yaml"
+    images:
+      - registry.k8s.io/metrics-server/metrics-server:v0.6.4
```

Comment:
Should this Zarf package use flavors and provide an option for https://registry1.dso.mil/harbor/projects/3/repositories/opensource%2Fkubernetes-sigs%2Fmetrics-server/artifacts-tab ?

Reply:
Yeah I was thinking we could come do all those at once since we need to change some stuff around how we do publishing / testing too, thoughts?

Reply:
I think it makes sense to approach flavoring all the things together/at the same time, since nothing else is flavored currently.